src/couch/src/couch_httpd.erl

```diff
@@ -252,7 +259,10 @@ absolute_uri(_Req, _Path) ->
     throw({bad_request, "path must begin with a /."}).
 
 unquote(UrlEncodedString) ->
-    mochiweb_util:unquote(UrlEncodedString).
```

Comment:
Would it work to use chttp:unquote/1 to avoid duplication?

Reply:
I have changed it to where `couch_httpd:unquote/1` calls `chttpd:unquote/1`.